File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/multipart-form-data.js
var unirest = require("unirest");

var req = unirest("POST", "http://mockbin.com/har");

req.headers({
  "content-type": "multipart/form-data; boundary=---011000010111000001101001"
});

req.multipart([
  {
    "body": "bar"
  }
]);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: packages/insomnia-httpsnippet/test/fixtures/output/c/libcurl/multipart-form-data.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "POST");
curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "content-type: multipart/form-data; boundary=---011000010111000001101001");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

curl_easy_setopt(hnd, CURLOPT_POSTFIELDS, "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--\r\n");

CURLcode ret = curl_easy_perform(hnd);
